gpflux/models/deep_gp.py

```diff
@@ -291,6 +311,12 @@ def as_prediction_model(
         return model_class(self.inputs, outputs)
 
 
+class OrthDeepGP(DeepGP):
+    """
+    Orthogonal Deep Gaussian Processes
+    """
+
+
```

Comment:
This is also declared in the `gpflux.models.orth_deep_gp` module.

Reply:
ahh right, forgot to delete that file.